sections/preliminaries.tex

```diff
@@ -36,11 +36,24 @@ \subsection{\Acrlong{lda}}
 
 
 \subsection{Author-topic \gls{lda}}\label{subsec:auth_prelim} \vejleder{introducer modellen som den er i \cite{author_topic_2012}, gerne flere detaljer}
-\todo[inline]{Beskriv den generative process}
 \citet{author_topic_2012} present the author-topic model.
 It seeks to find topics based on author metadata, and is based on the assumption that authors prefer to write about specific topics.
 In this model, there are no document-topic distributions $\theta$, but rather one author-topic distribution for each author.
-
+The reason for this is to find the interest of authors, within the documents, that we are analyzing.
+The generative process for the author-topic model is similar to that of the \gls{lda} with a few minor changes.
+The model assumes that there are multiple authors for each document which is modeled by the $x$ variable in \autoref{fig:original_author_lda}
+Before drawing a word topic, we need to select an author before drawing a topic which is executed in B.a.
+\vspace{\topsep}
+\begin{enumerate}
+	\item Draw topic proportion $\theta_a \sim Dirichlet(\alpha)$
+	\item For each word $n$ in the document:
+	\begin{enumerate}
+		\item Draw author assignment $x \sim \boldsymbol{a_d}$
+		\item Draw topic assignment $z_{d,n} \sim Mult(\theta_a)$
```

Comment:
```suggestion
		\item Draw topic assignment $z_{d,n} \sim Mult(\theta_x)$
```
tror jeg vil bruge x her, for at understrege at det er den udtrukne author der bruges